CMakeLists.txt

```diff
@@ -47,21 +47,29 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Version tokens
-include(GetGitRevisionDescription)
-get_git_head_revision(GIT_REFSPEC NVIM_VERSION_COMMIT)
-if(NOT NVIM_VERSION_COMMIT)
-  set(NVIM_VERSION_COMMIT "?")
-endif()
 set(NVIM_VERSION_MAJOR 0)
-set(NVIM_VERSION_MINOR 0)
+set(NVIM_VERSION_MINOR 1)
 set(NVIM_VERSION_PATCH 0)
 set(NVIM_VERSION_PRERELEASE "-alpha")
-# TODO(justinmk): UTC time would be nice here #1071
-git_timestamp(GIT_TIMESTAMP)
-# TODO(justinmk): do not set this for "release" builds #1071
```

Comment:
I think I have changed my mind on this, unless others object. If git _is_ available, I don't see why _not_ collect a SHA and list it in `:version`.